MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/analytics/AnalyticsTracker.java

```diff
@@ -0,0 +1,177 @@
+package com.mapbox.mapboxandroiddemo.analytics;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.messages.IdentifyMessage;
+import com.segment.analytics.messages.ScreenMessage;
+import com.segment.analytics.messages.TrackMessage;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * This class abstracts various analytics calls to Segment analytics' Java library.
+ */
+
+public class AnalyticsTracker {
```

Comment:
Shared analytics code module renders the class above obsolete. It has been removed from the wearable demo app.